vowpalwabbit/cbify.cc

```diff
@@ -40,6 +40,8 @@ struct cbify
   MwtExplorer<example>* mwt_explorer;
   vw_recorder* recorder;
   v_array<action_score> a_s;
+  // used as the seed
+  size_t example_counter;
```

Comment:
Is this being initialized somewhere? I see it's just being used as a seed so bogus starting values are still ok, although someone else could use it for its name.